async-seq.js

```diff
@@ -0,0 +1,26 @@
+const fs    = require('fs');
+const path  = require('path');
+const async = require('async');
+const R = require('ramda');
+
+const join = (x, cb) => cb(null, path.join(process.argv[2], x));
+const map = R.curry(R.flip(async.map));
```

Comment:
I believe `R.flip` returns a curried function, so we needn't apply `R.curry` here.